tide/core/Content.h

```diff
@@ -105,6 +106,12 @@ class Content : public QObject
     /** Get the dimensions. */
     QSize getDimensions() const;
 
+    /** Get the width. */
+    int width() const;
```

Comment:
I like that naming more than getFoo(), but it's inconsistent to the rest of the functions. You plan to transition to another naming scheme?